warehouse/query-core/src/main/java/datawave/query/iterator/QueryOptions.java

```diff
@@ -1282,17 +1281,18 @@ public boolean validateOptions(Map<String,String> options) {
         
         if (options.containsKey(PROJECTION_FIELDS)) {
             this.projectResults = true;
-            this.useWhiteListedFields = true;
             
             String fieldList = options.get(PROJECTION_FIELDS);
-            if (fieldList != null && EVERYTHING.equals(fieldList)) {
-                this.whiteListedFields = UniversalSet.instance();
-            } else if (fieldList != null && !fieldList.trim().equals("")) {
+            if (fieldList != null && !EVERYTHING.equals(fieldList) && !fieldList.trim().equals("")) {
+                this.useWhiteListedFields = true;
```

Comment:
I'm a little confused on this one.  We used to use UniversalSet if the field list was *, now we don't do that.  We only set the whitelisted fields if we have a non-`*`, non-empty list of fields.  So I guess we're saying that is the field list is set to `*`, we want to instead go down the blacklisted fields path with an empty set of fields.

Reply:
That is correct.  The UniversalSet mechanism was actually messing things up.  So instead a whitelist of * is the equivalent of an empty blacklist.  Although I suppose I could change this to avoid applying a projection at all in that case.  I will change it to do that instead.